rootfs/etc/profile.d/_20-mounts.sh

```
@@ -0,0 +1,180 @@
+			return 9
+		fi
+		GEODESIC_HOST_PATHS+=("${src}/" "${dest}/")
+		if [[ -z $GEODESIC_HOST_UID ]] || [[ -z $GEODESIC_HOST_GID ]]; then
+			red '# ERROR: `$MAP_FILE_OWNERSHIP` is set to "true" but `$GEODESIC_HOST_UID` and `$GEODESIC_HOST_GID` are not set.'
+			red '# File ownership mapping will not be enabled.'
+			findmnt -fn "${dest}" >/dev/null ||
+				mount --rbind "${src}" "${dest}"
+		else
+			green "# File ownership mapping enabled."
+			green "# Files created on host will have UID:GID ${GEODESIC_HOST_UID}:${GEODESIC_HOST_GID} on host."
+			local bindfs_opts=(-o nonempty ${GEODESIC_BINDFS_OPTIONS} "--map=${GEODESIC_HOST_UID}/0:@${GEODESIC_HOST_GID}/@0")
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)